go.mod

```diff
@@ -1,113 +1,115 @@
 module github.com/conductorone/baton-crowdstrike
 
-go 1.20
+go 1.22
 
 require (
-	github.com/conductorone/baton-sdk v0.1.11
-	github.com/crowdstrike/gofalcon v0.4.2
+	github.com/conductorone/baton-sdk v0.1.33
+	github.com/crowdstrike/gofalcon v0.3.1
```

Comment:
It looks like a release candidate has the fix: https://github.com/CrowdStrike/gofalcon/releases/tag/0.7.0-rc1

Do we want to try using that? If it works, we can switch to 0.7.0 when it comes out.

Reply:
Seems like the github release went through, but I can't find it in the go registry: https://pkg.go.dev/github.com/crowdstrike/gofalcon?tab=versions. Will let them know.

Reply:
Was not aware of installation steps of pre-releases, fixed in 84dacf5406f9ef014b1bd5a1662c06b267e221e9